jvb/src/main/kotlin/org/jitsi/videobridge/cc/allocation/BitrateController.kt

```diff
@@ -104,13 +103,17 @@ class BitrateController<T : MediaSourceContainer> @JvmOverloads constructor(
         eventEmitter.addHandler(eventHandler)
     }
 
+    private var bweSet = false
+
     /**
-     * Ignore the bandwidth estimations in the first 10 seconds because the REMBs don't ramp up fast enough. This needs
+     * Ignore the bandwidth estimations for some initial time because the REMBs don't ramp up fast enough.
+     * This is needed for our GCC implementation but should be ski
```

Comment:
The original comment is already confusing, the sentence terminating mid-word doesn't help :)